### PR TITLE
commit

### PR DESCRIPTION
**Pull request by nirvanashub**
check out my programs

### CODE REVIEW

File: variables/day2/day3/18/2/22 function/arrowfunc2.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// if only one statement is there, then () and return keyword can be avoided
+
+const division = (a,b) => a/b
+console.log(division(25,5));
+
+const square = num => num*num
```
Comment (Nilesh28v) [inline variables/day2/day3/18/2/22 function/arrowfunc2.js]: you have written same code for square and multiplication .

File: variables/day2/day3/18/2/22 function/filterfunc.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const arr = [1, 5, 6, 8, 9, 7];
+
+const result = arr.filter((num) => {
+    return num < 9;
+});
+console.log(result);
+
+// optimized code
+const result2 = arr.filter(num => num > 5);
```
Comment (Nilesh28v) [inline variables/day2/day3/18/2/22 function/filterfunc.js]: i think you can remove ( )  this also you can check .

File: variables/day2/day3/18/2/22 function/map.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const arr = [5,2,3,9,8,7,4];
+const resultarr = arr.map((num, index, aaray) => {
```
Comment (Nilesh28v) [inline variables/day2/day3/18/2/22 function/map.js]: remove index and aaray and then run i think there will be no change.

**Review by Nilesh28v — APPROVED** [review]

File: variables/day2/day3/18/2/22 function/arrowfunc2.js (hunk 1 above)
Comment (Rahul7808) [reply variables/day2/day3/18/2/22 function/arrowfunc2.js]: add comment

File: variables/day2/day3/18/2/22 function/map.js (hunk 3 above)
Comment (nirvanashub) [reply variables/day2/day3/18/2/22 function/map.js]: okay nilesh

**Comment by nirvanashub** [comment]
resolved

**Pull request merged**